### DISCUSSION TITLE
Should Youtube embeds auto-enable captions?

### DISCUSSION THREAD

**Discussion by alismith-dev** [CodeYourFuture/curriculum#230] [Ideas]
Pretty much what the title says: should we enable captions in Youtube embeds by default?

Some trainees struggle with (technical) English, and my assumption is that captions would help with understanding. (But noting that this is an assumption and happy to be corrected). For those that don't want them, it's easy to click the button in the toolbar to disable them.

I think the main risk is that AI-generated captions - which are likely to be the majority - may be pretty useless or potentially actively misleading.

From an implementation perspective, it appears that setting `cc_load_policy=1` in the embed query params should enable them by default, [see docs](https://developers.google.com/youtube/player_parameters#Parameters)

**Comment by SallyMcGrath** [comment]
Yes please - auto captions please. The captions are mainly good. 

I actually have my browser set to [auto live caption all video](https://support.google.com/chrome/answer/10538231?hl=en), which is better because it's a separate floating window so you can switch tabs. I recommend it!

**Reply by alismith-dev** [reply]
OK thanks - I'll go ahead and convert this to a ticket then.
